src/main/java/AddCommand.java

```diff
@@ -0,0 +1,20 @@
+public class AddCommand extends Command{
+    private Task myTask;
+    public AddCommand(Task myTask) {
+        this.myTask = myTask;
+    }
+
+    /**
+     * Adds a task to toDoList in TaskList, saves the new task list in a
+     * txt file and prints a success message.
+     * @param ui a Ui instance to enable calling of Ui functions
```

Comment:
Perhaps you could add an empty line after the JavaDoc header comment to separate the header and the parameter description? Also, I think you might have to add a full stop after all sentences in JavaDoc comments? I observed a similar issue in other JavaDoc comments.